testing/php7-ssh2/APKBUILD

```diff
@@ -0,0 +1,33 @@
+# Contributor: Andy Postnikov <apostnikov@gmail.com>
+# Maintainer: Valery Kartel <valery.kartel@gmail.com>
```

Comment:
He is maintainer of php5-ssh2 but better to ask him
Probably I need to apply for mainteinership somehow